src/main/java/Task.java

```diff
@@ -0,0 +1,49 @@
+import java.time.LocalDate;
```

Comment:
Javadocs not present in some classes and methods. While there are some comments for methods and classes, the format should look something like this:

```java
/**
 * Calculates the area of a rectangle.
 * 
 * @param width The width of the rectangle. Must be greater than zero.
 * @param height The height of the rectangle. Must be greater than zero.
 * @return The area of the rectangle calculated using the formula {@code width * height}.
 * @throws IllegalArgumentException if either {@code width} or {@code height} is less than or equal to zero.
 */
public double calculateRectangleArea(double width, double height) {
    if (width <= 0 || height <= 0) {
        throw new IllegalArgumentException("Width and height must be greater than zero.");
    }
    return width * height;
}

```